lib/ansible/modules/async_status.py

```diff
@@ -28,6 +28,8 @@
     type: str
     choices: [ cleanup, status ]
     default: status
+notes:
+  - In ansible-core 2.19, return fields that were using 0/1 (int) were updated to be 'false'/'true' (bool) values instead.
```

Comment:
```suggestion
  - The RV(started) and RV(finished) return values were updated to return V(True) or V(False) instead of V(1) or V(0) in ansible-core 2.19.
```